src/dns/rfc1035.cc

```diff
@@ -109,7 +109,8 @@ rfc1035LabelPack(char *buf, size_t sz, const char *label)
     assert(sz >= len + 1);
     *(buf + off) = (char) len;
     off++;
-    memcpy(buf + off, label, len);
+    if (label)
+        memcpy(buf + off, label, len);
```

Comment:
This change conflicts with #1613 that has been cleared for merging already. If you more-or-less agree with that PR changes, please refactor your PR to exclude rfc1035LabelPack() fix (including updating PR title and description as they will be used as the future official [commit message](https://github.com/measurement-factory/anubis/blob/master/README.md#commit-message)). Otherwise, we should probably block #1613 from merging.

Reply:
That PR looks like a fine solution too. I'll refactor this PR.